app/controllers/stories_controller.rb

```diff
@@ -324,7 +324,7 @@ def assign_podcasts
   def redirect_to_lowercase_username
     return unless params[:username]&.match?(/[[:upper:]]/)
 
-    redirect_permanently_to("/#{params[:username].downcase}")
+    redirect_permanently_to(action: :index, username: params[:username].downcase)
```

Comment:
Instead of depending on raw string URL, this now call upon the specific action and provide a param.